src/components/settings.jsx

```diff
@@ -18,29 +18,28 @@ class Settings extends Component {
             bio: '',
             url: '',
             image: '', 
+            change: false
         };
     }
-
     showNotification = () => {
         new Noty({
             type: 'success',
             theme: 'bootstrap-v4',
-            layout: 'topRight',
+            layout: 'bottomRight',
             text: 'You Changes Have Been Saved!',
             timeout: 3000
         }).show()
     }
-   
     handleChange = e => {
         if(e.target.files[0]) {
             
             let image = e.target.files[0];
             console.log(image);
-            const uploadTask = storage.ref(`images/${image.name}`).put(image);
+            const uploadTask = storage.ref(`images/${this.props.googleId}`).put(image);
             uploadTask.on('state_changed',
         
             () => {          
-                    storage.ref('images').child(image.name).getDownloadURL().then(url => {
+                    storage.ref('images').child(this.props.googleId).getDownloadURL().then(url => {
```

Comment:
should we make the path 'images/${this.props.googleId}/profilePic`)
Cuz we will have to store multiple images under the user id